adage/analyze/management/commands/import_data.py

```diff
@@ -22,7 +25,8 @@
 #        repository into `adage-server` or (b) make the code we need into a
 #        pip-installable utility package that gets imported by both the `ADAGE`
 #        project and the `adage-server` project.
-sys.path.append(os.path.abspath('../../'))
+repo_parent = os.path.dirname(os.path.dirname(settings.BASE_DIR))
+sys.path.append(os.path.abspath(repo_parent))
```

Comment:
Oh, drat. Of course the deployment pulled from the official repo, not this branch, so this was not a successful test. Trying again.